workday/12.1/modules/ROOT/pages/workday-connector-studio.adoc

```diff
@@ -0,0 +1,132 @@
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with Workday Connector:
+
+* On Modified Objects +
+Starts your app when an object is created, modified, or deleted in Workday.
+* HTTP Listener +
+Starts your app using a browser or a command such as `curl`.
+* Scheduler +
+Starts your app at specified timed intervals.
+
+To configure an *On Modified Objects* input source:
+
+. In the *Mule Palette* view, select *Workday > On Modified Objects*.
+. Drag *On Modified Objects* to the Studio canvas.
+. On the *On Modified Objects* configuration screen, optionally change the value of the *Display Name* field.
+. Enter the required information for the following fields:
+* *Object type* 
+* *Scheduling Strategy* +
+Tells the source how it can poll the data. This field accepts two options:
+** *Fixed Frequency*
++
+Polls for data at a specified number of milliseconds, seconds, minutes, hours, or days. The default is 1000 milliseconds.
+** *Cron*
++
+Accepts an expression that polls for data based on a filter, such as, every Monday in a month at a given time. See xref:mule-runtime::scheduler-concept.adoc#cron-expressions[Cron Expressions] for examples.
++
+. Click the plus sign (+) next to the *Connector configuration* field to <<configure-global-element,configure a global element>> that can be used by all instances of the On Modified Objects operation in the app.
+
+
+[[add-connector-operation]]
+== Add a Connector Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform. Workday Connector operations enable your app to process data received from the input source. 
+
+To add an operation for Workday Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Workday* and then select the desired operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+Each Workday operation has its own required fields in Anypoint Studio. For more information, see
+the xref:workday-reference.adoc[Connector Reference].
+
+[[configure-global-element]]
+== Configure a Global Element
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. Configuring a global element requires you to provide the authentication credentials the connector uses to access the target Workday system. 
+
+To configure the global element for Workday Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the configuration screen for the operation, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *General* tab, configure the authentication for the connector:
++
+[%header,cols="30s,70a"]
+|===
+|Field |Description
```

Comment:
I want to try to format the Description column to follow bullet-list formatting rules, so that we can standardize to fragments optionally followed by sentence in a new para. See below.